src/fhir/resources.ts

```diff
@@ -105,7 +105,7 @@ export const organisationAndId = (form: AddressSchema): ResourceAndIds => {
   org.name = form.name;
   org.address = [
     {
-      line: form.streetAddress,
+      line: [form.streetAddress], // array type to conform with FHIR requirements
```

Comment:
I think with these we should split the string by `,` so that it becomes an array. As this is nominally lines of addresses and comma separation is fairly standard